src/store/RootState.ts

```diff
@@ -18,7 +18,13 @@ export interface ItemValue {
 
 export type StringValue = string;
 
-export type Value = ItemValue | StringValue | null;
+export interface QuantityValue {
+	value: number;
+	precision?: number;
+	unit: ItemValue;
```

Comment:
The unit should either be also optional or it should be nullable.

Reply:
adressed